### ISSUE TITLE
NAF in LAN 

### ISSUE THREAD

**Issue by jimver04** [networked-aframe/networked-aframe#377]
Hi Vincent and NAF community, 

I want to run NAF in a local network in a public library, i.e. in 192.168.0.100:5832 where library visitors will enter with their mobile phones and their avatars will be their video streams in the VR space. 

There are two issues as can be seen in the following image
![image](https://user-images.githubusercontent.com/1898502/202086686-7edb57bc-72ce-4cef-b1c8-b35766d6e3de.png)

1) Aframe can not access the microphone and camera (the message in the white screen in the viewport); and 
2) NAF can not connect because it can not access the video stream I guess:  "TypeError: Cannot read properties of undefined (reading 'getUserMedia')" at NAFLogger.js L37 as in the image below. This might be a consequence of the first issue
![image](https://user-images.githubusercontent.com/1898502/202087203-39f6e3e0-3706-4f5c-b158-dd0da36abd29.png)

I do not have any idea how to solve this. Any ideas ?

Best,
Dimitrios

**Comment by jimver04** [comment]
I am pasting some initial research for candidate solutions

###  1:  "Self-Signed certificate" -> Rejected

According to this below a self-signed certificate is not possible to be installed in an non-rooted Android device:
 
https://security.stackexchange.com/questions/121163/how-do-i-run-proper-https-on-an-internal-network?newreg=455b4cb0bb0c44a7999eed629560bf4d

**Comment by jimver04** [comment]
I think disable chrome browser ssl is the only solution to go.

**Comment by vincentfretin** [comment]
Yes `getUserMedia` is only available on secure context so you need https.
You need to create a self signed certificate on your machine where you host the experience.
The webpack-dev-server with https enabled creates one automatically for example. If you use the easyrtc server, you need to create one yourself with mkcert or openssl cli and uncomment the lines in the `server/easyrtc-server.js` file.
On Android devices you can access the experience on https://192.168.1.15:8080 and you just need to accept the certificate and it works.
